File: services/objs-service/src/main/java/io/qpointz/poc/objs/service/ObjsServiceAutoConfiguration.java
package io.qpointz.poc.objs.service;

import org.springframework.boot.autoconfigure.AutoConfiguration;
import org.springframework.context.annotation.ComponentScan;

/**
 * Autoconfiguration entry for objs REST and service beans.
 */
@AutoConfiguration
@ComponentScan(basePackages = "io.qpointz.poc.objs.service")
public class ObjsServiceAutoConfiguration {
}

File: objs-service/src/main/kotlin/org/poc/objs/service/ObjsServiceAutoConfiguration.kt
package org.poc.objs.service

import org.springframework.boot.autoconfigure.AutoConfiguration
import org.springframework.context.annotation.ComponentScan

/**
 * Autoconfiguration entry for objs REST and service beans.
 */
@AutoConfiguration
@ComponentScan(basePackages = ["org.poc.objs.service"])
class ObjsServiceAutoConfiguration
